OWASP/SecureTea-Project
# Connected application to Sqlite to store logs

## Status
READY

## Description
This PR connects the application to Sqlite to store the logs.

## Todos


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

```sh
git pull --prune
git checkout feature-logging
sudo python setup.py
sudo SecureTea.py
```
You may check the entries in the database created at `/etc/securetea/`
```sqlite
cd /etc/securetea/
sqlite3 db.sqlite3
select * from LOGS;
```

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Logging
